applications/crossbar/src/modules/cb_cdrs.erl

```diff
@@ -113,6 +113,8 @@ to_json({Req, Context}) ->
 to_csv({Req, Context}) ->
     {Req, to_response(Context, <<"csv">>, cb_context:req_nouns(Context))}.
 
+-spec to_response(cb_context:context(), kz_term:ne_binary(), req_nouns()) ->
+                         {cb_cowboy_payload(), cb_context:context()}.
```

Comment:
It returns `cb_context:context()`

Reply:
So it does. Come on Dialyzer! You had one job!

Reply:
Actually, if I told dialyzer about cb_context, it would have known. And, in fact, including it in the mix gets an error reported. I'm sorry Dialyzer...